scripts/common/rook-upgrade.sh

```diff
@@ -281,6 +283,33 @@ function rook_upgrade_addon_fetch_and_load_airgap() {
     logSuccess "Images loaded for Rook $from_version to $to_version upgrade"
 }
 
+# rook_upgrade_has_all_addon_version_packages will return 1 if any add-on versions are missing that
+# are necessary to perform the upgrade.
+function rook_upgrade_has_all_addon_version_packages() {
+    local from_version="$1"
+    local to_version="$2"
+
+    if rook_upgrade_is_version_included "$from_version" "$to_version" "1.4" ; then
+        if [ ! -f "addons/rookupgrade/10to14/Manifest" ]; then
+            return 1
```

Comment:
Should this log that the package is missing?

I'm not 100% sure what the failure flow case is here

Reply:
this function is called in an if statement. if 1 is returned then the script will prompt to upload the package

```
if rook_upgrade_has_all_addon_version_packages "$from_version" "$to_version" ; then
```